.travis.yml

```diff
@@ -59,10 +59,11 @@ deploy:
   user: bbp.opensource
   password:
     secure: nItO6zvjqabsDOVyqj4k4V7dJXKz5FN9mkzlunTpZjJDQKISyN+lfDbrpjKihZyToY4fP5XsPnDK6zInnVFmjLDDrAsNh4IaNdU10HSmvGpTxCr3ecAMnLrBHHQrs7LplsjorSFwGjpaGr+M0PD3axi+itiMtTnuwaVH4CS0ywKMqt5575Wn8hdgJ7x/OglbA16DjhyoupdKGBdSv7HhB7AujOjQOfbnTWjuMVcjF3FFoARZuKdlmyDybMsujwgaTUxSLBVE8FqRifcukCAAmTn1O3UYNOY53QK0NsPYNvjRWAK9S/uMzOTduX69zDl7ChBtrsaplfdpqGv4U4QagIr5YgdnxVAyvLWsBOf+vaZFT2NwseHQPGHp5ftlAwYj0xk/3bJVINa1cgnsK9p+GErrBCI42Bqg5N/O/dhqnuB2136gPW8Ph4Jd6yINO58NImQ6ofC2Jg35MdNb5G9ceBqn5j+77ze1eLPwlaOTDZ9g5c1GzzUx6RPrzxP8OUqLctxgXdOaIUGFyet4raeP/aeGeJGndfdCbhhjjbSeLGH2se/HFd1y57YtMfTmYSKcy2PHqe1g9kWqrBsbOLzsaDO6dme56J2HZWTv0Qwz7f2scs9Lx6OgT0g9JU02vieVrjfzPmhqoT4rQ+D5ssCW5Qv34ygJepKHD4AIWWciitc=
-  skip_cleanup: true
+  cleanup: false
```

Comment:
I recommend to keep the skip_cleanup= true (see https://docs.travis-ci.com/user/deployment/pypi/#upload-artifacts-only-once). Otherwise looks good to me.

Reply:
I replaced with `cleanup: false`, because of the travis warning:
`deploy: deprecated key skip_cleanup (not supported in dpl v2, use cleanup)`

Reply:
there is a release note about this issue: https://blog.travis-ci.com/2019-08-27-deployment-tooling-dpl-v2-preview-release

Reply:
I see. From travis doc the dpl v2 is still beta and v1 is still the one used by default hence my comment. If we want to use dpl v2 it looks like w should add some extra directives: 

See here. https://docs.travis-ci.com/user/deployment-v2#cleaning-up-the-git-working-directory

I guess my point is more like is 'cleanup' a valid directive in dpl v1 ?

Reply:
I thought we get this warning precisely because they switched to v2, otherwise I don't think it works for v1 :(

Reply:
I think v1 is till used as per the header note of this document: https://docs.travis-ci.com/user/deployment-v2.

If you want to use already the dpl v2 I recommend to add in .travis.yml the directives suggested in: https://docs.travis-ci.com/user/deployment-v2#how-to-opt-in-to-v2

Reply:
Alright, I put back `skip_cleanup: True`

Reply:
Okay for me.